documentation/src/main/wiki/Dictionary.md

```diff
@@ -13,20 +22,15 @@ a set of system modules that are crucial for the whole system to work. The AET s
 implementations of jobs that can perform a particular task (e.g. collect screenshots, compare sources, validate a page against *W3C HTML5*).
 
 *AET Maven Plugin*  
-a default client application for the AET system that is used to trigger the execution of the *Test Suite*.
-
-*Amazon Web Services*  
-cloud computing services where the AET environment can be deployed to.
+a default client application for the AET system that was used to trigger the execution of the *Test Suite*.
```

Comment:
Maven plugin is no longer default :)
It is just `a client application`.

Reply:
Good point! I missed that. done.